src/database/DatabaseConnectionHandler.java

```diff
@@ -1,58 +1,57 @@
 package database;
```

Comment:
Ignore all of this file. It's just reformatting from a plugin I have and removing Patty's user and password lol